powertools-large-messages/src/main/java/software/amazon/lambda/powertools/largemessages/LargeMessageConfig.java

```diff
@@ -0,0 +1,59 @@
+package software.amazon.lambda.powertools.largemessages;
+
+import software.amazon.awssdk.http.urlconnection.UrlConnectionHttpClient;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.s3.S3Client;
+import software.amazon.awssdk.services.s3.S3ClientBuilder;
+
+import static software.amazon.lambda.powertools.core.internal.LambdaConstants.AWS_REGION_ENV;
+
+/**
+ * Singleton instance for Large Message Config.
```

Comment:
Just to make it super explicit _why singleton_ ... 

```suggestion
 * Singleton instance for Large Message Config. We need this to provide a way to customize the S3 client configuration used by the annotation.
```